Makefile

```diff
@@ -65,12 +65,7 @@ build-integration: ## Build integration test binary
 
 .PHONY: test-e2e
 test-e2e: ## Run openshift specific e2e test
-	go test -timeout 60m \
-		-v ./vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
-		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
-		-machine-api-namespace $${NAMESPACE:-openshift-machine-api} \
-		-ginkgo.v \
-		-args -v 5 -logtostderr true
+	$(MAKE) -C ./vendor/github.com/openshift/cluster-api-actuator-pkg $@
```

Comment:
don't we want to pass `-ginkgo.v -ginkgo.noColor=true` here?
do we want to run it like https://github.com/openshift/cluster-api-actuator-pkg/blob/master/Makefile#L52-L53

Reply:
@enxebre it's doing exactly that, `$@` gets replaced to test-e2e which executes then from the file you referenced:
```sh
hack/ci-integration.sh -ginkgo.v -ginkgo.noColor=true -ginkgo.focus "Feature:Operators"
hack/ci-integration.sh -ginkgo.v -ginkgo.noColor=true -ginkgo.skip "Feature:Operators"
```